src/App.tsx

```diff
@@ -1,10 +1,45 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+
+import { useCallback, useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { filterTodos } from './utils/services';
+import TodoItem from './components/TodoItem';
+import Header from './components/Header';
+import Footer from './components/Footer';
+import ErrorNotification from './components/ErrorNotification';
 
 export const App: React.FC = () => {
+  const [loading, setLoading] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```

for boolean state better use is/has prefix like isLoading/hasError